### PR TITLE
Add var for artful ubuntu (17.10)

### PR DESCRIPTION
**Pull request by Edznux**

### CODE REVIEW

File: vars/ubuntu/artful [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (naftulikay) [inline vars/ubuntu/artful]: Can you please rename this file to include a `.yml` at the end of the path?

**Review by naftulikay — CHANGES_REQUESTED** [review]
Thanks so much for your contribution, and apologies for the delay.

**Comment by Edznux** [comment]
Ooops, updated !
First time using the ui to do a simple PR, failed it !

**Review by naftulikay — APPROVED** [review]

**Comment by naftulikay** [comment]
Cutting release 1.0.2 to include this. Thank you for your contribution!

Hopefully you've seen the other million roles too :smiley:

**Pull request merged**